showOutput3.js

```diff
@@ -935,19 +769,81 @@ document.getElementById('submit').addEventListener(
             "};"
         );
 
-
         /**************
         ** info.json **
         **************/
         var infojsonOutput = [
             "{",
             "    \"keyboard_name\": \""+ obj.keyboard.settings.name +"\",",
+            "    \"manufacturer\": \"Your Name Here\",",
             "    \"url\": \"\",",
-            "    \"maintainer\": \"qmk\",",
+            "    \"maintainer\": \"Your Name Here\",",
+            "    \"usb\": {",
+            "        \"vid\": \"0xFEED\",",
+            "        \"pid\": \"0x0000\",",
+            "        \"device_version\": \"1.0.0\"",
+            "    },",
+            "    \"processor\": \"" + MicroController + "\",",
+            "    \"bootloader\": \"atmel-dfu\",",
+            "    \"features\": {",
+            "        \"bootmagic\": " + (BootmagicEnable ? "true" : "false") + ",",
+            "        \"mousekey\": " + (MousekeyEnable ? "true" : "false") + ",",
+            "        \"extrakey\": " + (ExtrakeyEnable ? "true" : "false") + ",",
+            "        \"console\": true,",
+            "        \"command\": true,",
```

Comment:
The dev branch I did last year had both of these off by default, to save firmware space. Thoughts on that?